eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py

```diff
@@ -13,185 +13,125 @@
-                                       "release-species-inventory-table", "release-version", "species-release-folder"]),
-        ('update_sequence_names_to_ena', ["taxonomy-id", "assembly-accession", "species-release-folder",
-                                          'sequence-name-converter-path', 'bcftools-path']),
-        ("update_release_status_for_assembly", ["private-config-xml-file", "profile", "release-species-inventory-table",
-                                                "taxonomy-id", "assembly-accession", "release-version"])
-     ])
-
-workflow_process_template_for_nextflow = """
-process {workflow-process-name} {{
-
-    {cluster-options}
-    input:
-        val flag from {previous-process-output-flag}
-    output:
-        val true into {current-process-output-flag}
-    script:
-    \"\"\"
-    export PYTHONPATH={script-path} &&  ({python3-path} -m run_release_in_embassy.{process-with-args} 1>> {release-log-file} 2>&1)
-    \"\"\"
-}}
-"""
-
-
-def get_release_properties_for_current_species(common_release_properties, taxonomy_id):
-    release_properties = {"taxonomy-id": taxonomy_id,
-                          "species-release-folder": os.path.join(common_release_properties["release-folder"],
-                                                                 get_release_folder_name(taxonomy_id))}
-    os.makedirs(release_properties["species-release-folder"], exist_ok=True)
-    release_properties["timestamp"] = timestamp
-    release_properties["dump-dir"] = os.path.join(release_properties["species-release-folder"], "dumps")
-    os.makedirs(release_properties["dump-dir"], exist_ok=True)
-    return release_properties
-
-
-def get_release_properties_for_current_assembly(species_release_properties, assembly_accession):
-    release_properties = {
-        "assembly-accession": assembly_accession,
-        "assembly-release-folder": os.path.join(species_release_properties["species-release-folder"], assembly_accession),
-        "release-log-file": "{species-release-folder}/{assembly_accession}/release_{taxonomy-id}_{assembly_accession}_{timestamp}.log".format(**species_release_properties, assembly_accession=assembly_accession)
+logger = logging_config.get_logger(__name__)
+
+
+def get_nextflow_params(taxonomy_id, assembly_accession, release_version):
+    dump_dir = os.path.join(get_species_release_folder(taxonomy_id), 'dumps')
+    release_dir = get_assembly_release_folder(taxonomy_id, assembly_accession)
+    config_param = os.path.join(release_dir, f'nextflow_params_{taxonomy_id}_{assembly_accession}.yaml')
+    os.makedirs(dump_dir, exist_ok=True)
+    yaml_data = {
+        'assembly': assembly_accession,
+        'dump_dir': dump_dir,
+        'executable': cfg['executable'],
+        'jar': cfg['jar'],
+        'log_file': get_release_log_file_name(taxonomy_id, assembly_accession),
+        'maven': cfg['maven'],
+        'python_script': cfg.query('python', 'interpreter'),
```

Comment:
I think this is accessed from the `executable` section... `params.executable.python.interpreter` in the nextflow

Reply:
Removed